modules/services/emacs.nix

```
@@ -82,13 +82,13 @@ in {
     };
 
     startWithUserSession = lib.mkOption {
-      type = lib.types.bool;
+      type = lib.types.enum [ false true "graphical" ];
```

Actually I feel this is a bit better:

```suggestion
      type = with lib.types; either bool (enum [ "graphical" ]);
```